src/services/goToDefinition.ts

```diff
@@ -246,7 +249,9 @@ namespace ts.GoToDefinition {
         // There are cases when you extend a function by adding properties to it afterwards,
         // we want to strip those extra properties.
         // For deduping purposes, we also want to exclude any declarationNodes if provided.
-        const filteredDeclarations = filter(symbol.declarations, d => d !== declarationNode && (!isAssignmentDeclaration(d) || d === symbol.valueDeclaration)) || undefined;
+        const filteredDeclarations =

```

Comment:
Is this just a whitespace change?

Reply:
Yeah, sorry, late night push, and I forgot to add a comment about this just breaking the long line.  I'll remove it if you prefer not touching other parts.